src/components/Cards/GuildsCard/SortingButton.tsx

```
@@ -0,0 +1,39 @@
+
+interface SortingButtonProps {
+  children: ReactNode;
+  column: Column<GuildEntry, unknown>;
+}
+
+export const SortingButton = ({ children, column }: SortingButtonProps) => {
+  const [sortingState, setSortingState] = useState(0);
+  const handleCircularSorting = () => {
+    switch (sortingState) {
+      case 0:
+        column.toggleSorting(true);
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)